package.json

```diff
@@ -20,11 +20,12 @@
   "author": "Augusto Lemble <me@augustolemble.com>",
   "license": "ISC",
   "dependencies": {
+    "abi-decoder": "^1.0.8",
     "ethereumjs-abi": "^0.6.4",
     "protobufjs": "^6.0.2",
     "solc": "^0.4.9",
     "truffle-hdwallet-provider": "0.0.3",
-    "zeppelin-solidity": "^1.0.4"
+    "zeppelin-solidity": "1.0.4"
```

Comment:
I think we need a separated PR if we upgrade zeppelin-solidity, but not to continue using the same version. And we will need a new PR to update zeppelin solidity.